### ISSUE TITLE
[2.2.3] Clear if value is zero feature is broken

### ISSUE THREAD

**Issue by ankurk91** [kevinongko/vue-numeric#42]
Hi,

## Expected Behavior
If v-model value is `0.00` then input field should clear on focus

## Actual Behavior
Input remains same

## Steps to Reproduce the Problem
http://jsfiddle.net/cvg6c4r3/6/

I have also tried adding a directive but no luck
```js
Vue.directive('clear-zero', {
  bind: function (el, binding, vnode) {
    el.onfocus = function () {
      if (parseFloat(el.value) === 0) {
        console.log('Clear value');
        el.value = null;
      }
    };
  }
});
```
```html
<vue-numeric v-clear-zero currency="$" :precision="2" v-model.number="money"></vue-numeric>
```

## Specifications

  - Plugin Version: 2.2.3+
  - Vue.js Version: 2.5.2/2.4.4
  - Browser: chrome 62 beta
  - OS: macOs sierra

Ref https://github.com/kevinongko/vue-numeric/issues/26

**Comment by kevinongko** [comment]
Hi @ankurk91, 

Sorry for the returning issue. Should be fixed in v2.2.6

**Comment by shivanaru** [comment]
Is this fixed? I am still seeing it on v2.6.10
